e2e/awslogs_test.go

```diff
@@ -19,25 +19,35 @@ const (
 	awslogsRegionKey              = "--awslogs-region"
 	awslogsStreamKey              = "--awslogs-stream"
 	awslogsGroupKey               = "--awslogs-group"
+	awslogEndpointKey             = "--awslogs-endpoint"
 	testEcsLocalEndpointPort      = "51679"
 	testAwslogsCredentialEndpoint = ":" + testEcsLocalEndpointPort + "/creds"
-	testAwslogsRegion             = "us-west-2"
-	testAwsLogsStream             = "test-stream"
-	testAwsLogsGroup              = "test-shim-logger"
+	testAwslogsRegion             = "us-east-1"
+	testAwslogsStream             = "test-stream"
+	testAwslogsGroup              = "test-shim-logger"
+	testAwslogsEndpoint           = "http://localhost:4566"
```

Comment:
Looks like `4566` is the local stack default port. Can you add a comment to mention that so it's not a magic number? would helps those not familiar with localstack

I see the docs are here https://docs.localstack.cloud/getting-started/faq/#is-using-localhostlocalstackcloud4566-to-set-as-the-endpoint-for-aws-services-recommended

and they recommend `localhost.localstack.cloud` as endpoint - have you tried?

Reply:
Good call. It works.